js/cli/src/namespace.ts

```diff
@@ -1,39 +1,44 @@
 #!/usr/bin/env ts-node
-import { Wallet, CLI } from "@raindrop-studios/sol-command";
-import { AnchorProvider, web3 } from "@project-serum/anchor";
+import { CLI } from "@raindrop-studios/sol-command";
+import * as anchor from "@project-serum/anchor";
+import { web3 } from "@project-serum/anchor";
 import log from "loglevel";
-
-import { NamespaceProgram } from "@raindrops-protocol/raindrops";
+import {
+  IDL as NamespaceProgramIDL,
+} from "../../idl/namespace";
```

Comment:
Turbo repo pr allows us to import these from the package now

Reply:
Okay, I need to learn about Turbo, I dont quite understand what you mean, I think since this isn't in master I would need to have a follow up with the path changes?